portal/index.php

```diff
@@ -51,7 +51,7 @@
 $landingpage = "index.php?site=" . urlencode($_SESSION['site_id']);
 $logoService = new LogoService();
 $logoSrc = $logoService->getLogo("portal/login/primary");
-$logo2ndSrc = $logoService->getLogo("portal/login/secondary", "logoHSE.png"); /*rm - add secondary logo */
+$logo2ndSrc = $logoService->getLogo("portal/login/secondary"); /*rm - add secondary logo */
```

Comment:
So if the secondary logo is turned on it will show a broken image link (src="").  I think that behavior is ok, people can just go and uncheck the config option if the logo is missing.

Reply:
thanks @adunsulag for taking it into the patch

and it works ok if secondary is turned on but either the .png file or indeed the directory 'secondary' are missing - no error reports and there is simply no secondary logo displayed on the login screen.

Reply:
if i understand what you are saying: i tested with secondary logo turned on then deleted the logo file, then the secondary folder. both times this caused no problem or error message.